### PR TITLE
[integration] new: added script for comdirac to be run during certification tests

### PR DESCRIPTION
**Pull request by marianne013**
BEGINRELEASENOTES

*tests
NEW: added script for comdirac hackathon certification tests

ENDRELEASENOTES

### CODE REVIEW

**Comment by marianne013** [comment]
Addresses part of #6746.

**Comment by marianne013** [comment]
@fstagni I made a matching card on the trello board.

File: tests/System/client_shortcommands.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+group_name = dteam_user
+home_dir = /dteam/diracCertification
+default_se = UKI-LT2-IC-HEP-disk
+
+EOF
+
+# run dls command
+# this should list the content of home_dir
+echo "Testing dls command."
+dls
```
Comment (fstagni) [inline tests/System/client_shortcommands.sh:21]: Only this one? What about the other commands?

**Comment by marianne013** [comment]
@fstagni or possible  @chrisburr I don't understand why this test fails (at least I can't see any error). Do you know what went wrong ?

**Comment by fstagni** [comment]
> @fstagni or possible @chrisburr I don't understand why this test fails (at least I can't see any error). Do you know what went wrong ?

I know why, it's just because of https://github.com/DIRACGrid/DIRAC/issues/6472. I have restarted it.

**Review by fstagni — APPROVED** [review]

**Pull request merged**